packages/esm-patient-immunizations-app/src/immunizations/immunizations-form.component.tsx

```diff
@@ -119,14 +132,40 @@ const ImmunizationsForm: React.FC<DefaultWorkspaceProps> = ({ patientUuid, close
     updateSingle('currentDose', currentDose);
   };
 
+  const vaccines = mockPaginatedImmunizations;
+  const VaccineSelect = () => (
+    <Select
+      id="vaccine"
+      invalidText="Required"
+      labelText=""
```

Comment:
I think it would be preferable to put the "Vaccine" string here rather than having it as part of a separate `span` tag. The label text is semantically connected to the field it labels in a way that the span is not, which makes it better for most use-cases.